js/views/userPageVw.js

```diff
@@ -303,10 +303,19 @@ UserPageVw = pageVw.extend({
             model.set('headerURL', self.options.userModel.get('serverUrl') + "get_image?hash=" + model.get('profile').header_hash + "&guid=" + self.pageID);
             model.set('avatarURL', self.options.userModel.get('serverUrl') + "get_image?hash=" + model.get('profile').avatar_hash + "&guid=" + self.pageID);
           }
+
           // Cache user avatar in localStorage
           var profile = model.toJSON().profile;
           window.localStorage.setItem("avatar_" + self.pageID, profile.avatar_hash);
 
+          if (response.profile.website) {
+            var website = profile.website;
+            var parse_url_regex = /^(?:([A-Za-z]+):)?(\/{0,3})([0-9.\-A-Za-z]+)(?::(\d+))?(?:\/([^?#]*))?(?:\?([^#]*))?(?:#(.*))?$/;
+            var parts = parse_url_regex.exec(website);
+            profile.short_website = parts[3];
```

Comment:
My bad, I forgot to link to the relevant issue: https://github.com/OpenBazaar/OpenBazaar-Client/issues/575

This strips off the protocol and the file path, so you are just left with the domain name.